src/runtime/crt/host/CMakeLists.txt.template

```diff
@@ -0,0 +1,59 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
since additional stuff is added beyond what's here, and nothing is really templatized, what do you think about calling this CMakeLists.txt.prelude or something?

Reply:
makes sense, that would also make it clear that this file is not supposed to be used in this directory.

Reply:
I'd say we keep it as `.template` to match with the rest. Also, it could be templated in future if we want to set <CRT_LIBS> instead of building all libraries.